tt_helpers_test.go

```diff
@@ -0,0 +1,474 @@
+package tupleconv
```

Comment:
Please, test only the public API.
```suggestion
package tupleconv_test
```

Reply:
Fixed.